src/ol/tilegrid/tilegrid.js

```diff
@@ -162,6 +201,15 @@ ol.tilegrid.TileGrid.prototype.forEachTileCoordParentTileRange =
 
 
 /**
+ * Get the extent for this tile grid, if it was configured.
+ * @return {ol.Extent} Extent.
```

Comment:
I am pretty sure the need will come eventually. But you don't have to do it as part of this PR if you are not sure it is useful.

FWIW, I know I'll need `getSizes` on `ol.TileGrid.WMTS` (for creating MapFish Print spec objects).